charts/portainer/templates/service.yaml

```diff
@@ -22,8 +22,13 @@ spec:
       {{- if (and (eq .Values.service.type "NodePort") (not (empty .Values.service.httpNodePort))) }}
       nodePort: {{ .Values.service.httpNodePort}}
       {{- end }}
+      {{- if (eq .Values.service.type "NodePort") }}
```

Comment:
I'm a bit confused about the condition here, do we still need to differentiate the `.Values.service.edgeNodePort` and `.Values.service.edgePort` variables?

Reply:
If there are still cases where we'd want edgePort to be 8000 (like in a LB deploy), then yes :)

Reply:
Indeed.